doc/reference/cmdline.rst

```diff
@@ -548,4 +548,124 @@ Scaffolding is available via the :command:`odoo-bin scaffold` subcommand.
 
     $ odoo_bin scaffold my_module /addons/
 
-This will create module *my_module* in directory */addons/*.
\ No newline at end of file
+This will create module *my_module* in directory */addons/*.
+
+
+Cloc
+====
+
+.. program:: odoo-bin cloc
+
+Odoo Cloc is a tool to count the number of relevant lines written in
+python, javascript or xml. This can be used as a rough metric for pricing
+maintenance of customizations.
+
+It has two modes of operation, either by providing a path:
+
+.. option:: --path PATH, -p PATH
+
+Or by providing the name of a database:
+
+.. option:: --database DATABASE, -d DATABASE
+
+The addons path should be provided as well
+
+.. code-block:: console
+
+    $ odoo-bin cloc --addons-path=dirs -d database
+
+In the latter mode, only the custom code is accounted for.
+You can also use a configuration file -c configuration.conf instead of the addons-path
+
+.. code-block:: console
+
+    $ odoo-bin cloc -c configuration.conf -d database
+
+both mode can be used simultaneously to count the line for a given folder
+in addition of the lines in the given database.
+
+.. code-block:: console
+
+    $ odoo-bin cloc --addons-path=dirs -d database -p module_path
+
+
+The verbose option show the details per files.
+
+
+.. option:: --verbose, -v
+
+
+Which files Odoo Cloc counts ?
+------------------------------
+
+Database mode
+'''''''''''''
```

Comment:
```suggestion
.. _reference/cmdline/cloc/database-option:

With the :option:`--database` option
''''''''''''''''''''''''''''''''''''
```
To avoid implying that it excludes using `--path`